resources/qml/Settings/SettingView.qml

```diff
@@ -124,7 +125,7 @@ ScrollView
                 id: inheritStackProvider
                 containerStackId: Cura.MachineManager.activeMachineId
                 key: model.key
-                watchedProperties: [ "global_inherits_stack" ]
+                watchedProperties: [ "global_inherits_stack", "value" ]
```

Comment:
Why add value here? We already have a property provider that is looking at value. There is a pretty good reason (as documented above) to have a dedicated global_inherits_stack provider.